ci/copy-rest-spring.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e # fail fast
 
-if [[ ! "$1" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then 
+if [[ ! "$1" =~ ^v[0-9]+\.[0-9]+\.[0-9] ]]; then 
```

Comment:
be more specific: ^v[0-9]+\.[0-9]+\.[0-9]+(-SNAPSHOT)?$
And that "+" is supposed to stay there ;)